app/views/admin/course_subjects/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<% provide(:title, @course.name) %>
+<h1><%= @course.name %></h1>
+<h2><%= @course.description %></h2>
+<div class="center">
+  <div class="span6 offset3 btn-group">
+    <% @course.subjects.each do |subject| %>
+      <% if subject == @subject %>
+        <% type_button = "btn btn-large btn-primary" %>
+      <% else %>
+        <% type_button = "btn btn-large btn-default" %>
```

Comment:
if - else dùng toán tử 3 ngôi, cho đỡ lặp code nè,
vì khách nhau cái class default hay primary thôi mà